### ISSUE TITLE
Need to update disabled dates example in readme

### ISSUE THREAD

**Issue by utkarshdalal** [charliekassel/vuejs-datepicker#708]
The readme says the way to disable dates is 
`<datepicker :disabledDates="state.disabledDates"></datepicker>`

However, it should be `:disabled-dates` rather than `:disabledDates`. Spent many hours figuring this out, please update the docs.

**Comment by daninden** [comment]
I agree.. only spend 1 hour maybe.. then found your comment.. Thanks!

**Comment by iliyaZelenko** [comment]
https://github.com/charliekassel/vuejs-datepicker/pull/715
